content/en/security_platform/cloud_workload_security/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Cloud Workload Security
+kind: documentation
+---
+
+<div class="alert alert-warning">
+Cloud Workload Security is currently in beta. Contact <a href="https://docs.datadoghq.com/help/">Datadog support</a> for more information.
+</div>
+
+## Overview
+
+Datadog Cloud Workload Security (CSW) detects threats to your production workloads in real time. With Cloud Workload Security, monitor file and process activity across your environment to detect threats to your infrastructure, like AWS EC2 instances, and workloads, like Kubernetes clusters, in real time at the kernel level. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories on hosts or containers in real-time or **Process Execution Monitoring** to monitor process executions for malicious activity on hosts or containers in real time.
```

Comment:
```suggestion
Datadog Cloud Workload Security (CSW) detects threats to your production workloads in real time. With Cloud Workload Security, monitor file and process activity across your environment to detect threats to your infrastructure, like AWS EC2 instances, docker containers, or Kubernetes clusters, in real time at the kernel level. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories. Use **Process Execution Monitoring** to monitor process executions for suspicious, malicious, or anomalous activity.
```

Tried to remove some redundant terms, and fix our definition of infrastructure a bit